qtranslate_hooks.php

```diff
@@ -225,6 +225,9 @@ function qtranxf_wpseo_replacements($replacements){
 add_filter('link_description', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 add_filter('the_author', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 
+add_filter('get_terms', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
```

Comment:
Filter 'get_terms' is already in the code at line  484 in qtranslate_frontend.php. Did you test it, btw? Since terms are translated with a different method other than `useCurrentLanguageIfNotFoundUseDefaultLanguage` and I do not see how your filter can work.